docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -17,6 +17,14 @@ In the following example, the labels of the density selector are customized.
 
 {{"demo": "pages/components/data-grid/localization/CustomLocaleTextGrid.js", "bg": "inline"}}
 
+The default locale of Material-UI is English (United States). You can find all the locales supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/locales) in the GitHub repository.
+
+### Supported locales
+
+| Locale    | BCP 47 language tag | Import name |
+| :-------- | :------------------ | :---------- |
+| Bulgarian | bg-BG               | `bgBG`      |
+
```

Comment:
This is mean to go below `## Locale text`

Reply:
Ok, I assumed that it will still be "under construction" since it doesn't work with the `ThemeProvider`.